MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -55,6 +61,7 @@ vtkStandardNewMacro(vtkSlicerMarkupsToModelLogic);
 vtkSlicerMarkupsToModelLogic::vtkSlicerMarkupsToModelLogic()
 {
   ImportingScene=0;
```

Comment:
use this-> prefix for all member variables to distinguish them from local variables:
instead of 
ImportingScene = 0;
this->ImportingScene = 0;